Kentor.AuthServices/ClaimsIdentityExtensions.cs

```diff
@@ -45,14 +45,18 @@ public static Saml2Assertion ToSaml2Assertion(this ClaimsIdentity identity, Enti
         /// <param name="audience">Audience to set as audience restriction.</param>
         /// <param name="inResponseTo">In response to id</param>
         /// <param name="destinationUri">The destination Uri for the message</param>
+        /// <param name="sessionNotOnOrAfter">The time instant at which the session between the principal 
+        /// identified by the subject and the SAML authority issuing this statement
+        /// must be considered ended.</param>
         /// <returns>Saml2Assertion</returns>
         /// <exception cref="ArgumentNullException"></exception>
         public static Saml2Assertion ToSaml2Assertion(
-            this ClaimsIdentity identity,
-            EntityId issuer,
-            Uri audience,
-            Saml2Id inResponseTo,
-            Uri destinationUri)
+        this ClaimsIdentity identity,

```

Comment:
Please don't change indentation.

Reply:
Tried hard to not let my resharper formatting get involved, guess I missed it.